### PR TITLE
Bentoctl documentation for local dev with docker

### PR DESCRIPTION
**Pull request by v-rocheleau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -416,20 +416,61 @@ To work on the `bento_web` repository within a BentoV2 environment, run the foll
...
-which means on-the-fly Webpack building will be available.
+This will clone the `bento_web` repository into `./repos/web` if necessary, pull the dev image, and start the container in development mode, with a volume mapping to the `./repos/web` directory, which means on-the-fly Webpack building will be available.
+
+You can find the default image tag variables in `./etc/bento.env` and overwrite them in `local.env`, look for the 
+pattern `BENTOV2_[name]_VERSION`. 
+
+The version tags correspond to the PR **number** (not its name), e.g. `BENTOV2_WEB_VERSION=pr-216` indicates that the 
+image was built from PR #216 in bento_web. 
+
+Note: Most of the time, you will not need to worry about changing this, unless changes were made to the dev image's entrypoint.
```
Comment (davidlougheed) [inline README.md:427]: emphasize this with bold or something

File: README.md [hunk 2]
```diff
@@ -416,20 +416,61 @@ To work on the `bento_web` repository within a BentoV2 environment, run the foll
...
+
+By default, the images used are those built by github CI workflows, triggered by commit and PR events and published to the Bento images [registry](https://github.com/orgs/bento-platform/packages). If after changing the version tag of an image the service's container can no longer be created, it is probably because the tag does not exist on github.
+
+To remediate this, you have two options:
+- Create a PR for the branch you want to work on, in order to trigger a CI workflow that will build an image tagged with the PR number (**prefered**)
+- Manually build and tag a docker image on your machine (**avoid when possible**)
+
+**Local bento_web image example**
+
+Note: this approach is a last resort for local development only. In some situations, we cannot always assume that working CI artifacts are available for every service used by Bento.
```
Comment (davidlougheed) [inline README.md:439]: emphasize this with bolding

**Review by davidlougheed — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -416,20 +416,61 @@ To work on the `bento_web` repository within a BentoV2 environment, run the foll
 ./bentoctl.bash work-on web
 ```
 
-This will clone the `bento_web` repository into `./repos/web` if necessary, and start it in development mode,
-which means on-the-fly Webpack building will be available.
+This will clone the `bento_web` repository into `./repos/web` if necessary, pull the dev image, and start the container in development mode, with a volume mapping to the `./repos/web` directory, which means on-the-fly Webpack building will be available.
```
[the remark this answers is not in the document]
Comment (v-rocheleau) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (v-rocheleau) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (v-rocheleau) [reply README.md]: Done

File: README.md [hunk 4]
```diff
@@ -416,20 +416,61 @@ To work on the `bento_web` repository within a BentoV2 environment, run the foll
...
+
+You can find the default image tag variables in `./etc/bento.env` and overwrite them in `local.env`, look for the 
+pattern `BENTOV2_[name]_VERSION`. 
+
+The version tags correspond to the PR **number** (not its name), e.g. `BENTOV2_WEB_VERSION=pr-216` indicates that the 
+image was built from PR #216 in bento_web. 
+
+Note: Most of the time, you will not need to worry about changing this, unless changes were made to the dev image's entrypoint.
+
+**Where are the images?**
```
[the remark this answers is not in the document]
Comment (v-rocheleau) [reply README.md]: Done

File: README.md [hunk 5]
```diff
@@ -416,20 +416,61 @@ To work on the `bento_web` repository within a BentoV2 environment, run the foll
...
+
+By default, the images used are those built by github CI workflows, triggered by commit and PR events and published to the Bento images [registry](https://github.com/orgs/bento-platform/packages). If after changing the version tag of an image the service's container can no longer be created, it is probably because the tag does not exist on github.
+
+To remediate this, you have two options:
+- Create a PR for the branch you want to work on, in order to trigger a CI workflow that will build an image tagged with the PR number (**prefered**)
+- Manually build and tag a docker image on your machine (**avoid when possible**)
+
+**Local bento_web image example**
```
[the remark this answers is not in the document]
Comment (v-rocheleau) [reply README.md]: Done

File: README.md [hunk 6]
```diff
@@ -416,20 +428,62 @@ To work on the `bento_web` repository within a BentoV2 environment, run the foll
...
+pattern `BENTOV2_[name]_VERSION`. 
+
+The version tags correspond to the PR **number** (not its name), e.g. `BENTOV2_WEB_VERSION=pr-216` indicates that the 
+image was built from PR #216 in bento_web. 
+
+**Note: Most of the time, you will not need to worry about changing this, unless changes were made to the dev image's entrypoint.**
+
+### Where are the docker images?
+
+By default, the images used are those built by github CI workflows, triggered by commit and PR events and published to the Bento images [registry](https://github.com/orgs/bento-platform/packages). If after changing the version tag of an image the service's container can no longer be created, it is probably because the tag does not exist on github.
```
Comment (davidlougheed) [inline README.md:445]: text wrapping here too

**Review by davidlougheed — APPROVED** [review]

**Pull request merged**